html/pfappserver/lib/pfappserver/I18N/fr.po

```diff
@@ -12393,3 +12393,63 @@ msgid ""
 "IP address archive of the devices on your network when enabled (see "
 "Maintenance section)"
 msgstr "Archive les adresses IPs des appareils sur le réseau quand qu'activé (voir la section Maintenance)"
+
+# pf::action (SYSLOG_PARSER_REGEX_VIOLATION_ACTIONS)
```

Comment:
For the french translation we will use transifex for that. So for now you can revert your change on this file.